app/src/main/java/com/teamagam/gimelgimel/app/utils/NetworkUtil.java

```diff
@@ -96,7 +98,8 @@ public static InetAddress getIp() {
         return sIp;
     }
 
-    public static String getMac() {
-        return sMacAddress;
+    public static String getUserName(Context context) {
+        PreferenceUtil prefs = ((GGApplication) context.getApplicationContext()).getPrefs();
+        return prefs.getString(R.string.user_name_text_key,sMacAddress);
```

Comment:
It seems that a user name is always available since the user name preference has a default value (strings.pref_default_display_name).
If that's the situation, all the mac-address code isn't necessary, and can be safely deleted. !resolve